README.md

```diff
@@ -3,68 +3,29 @@
 Main language: `TypeScript`
 
 This project contains:
```

Comment:
I suggest to create directories in assets with name of md-file. For example, we have `architecture.md`. This file has references to images. So we create directory `assets/architecture` to group all the resources of the markdown.

Reply:
Done